docs/model-monitoring/view-mm-applications.md

```diff
@@ -0,0 +1,49 @@
+(view-mm-applications)=
+# View the model monitoring applications status in the UI
+
+The Monitoring app view provides you with a comprehensive overview of your model monitoring applications and their status, and an overview about the performance of the model monitoring infrastructure. This is where you can monitor the model monitoring. For example, if there is a lag in an app, this is the first place to get insights about what is causing the lag.
+
+You can choose the time period for the statistics. The default time period is 24 hours. The maximum time period is one month. The selected period operates as a sliding window of 24 hours, updated every hour.
+
+**In this section**
+- [Monitoring app view](#monitoring-app-view)
+- [Application page](#application-page)
+- [Application metrics](#application-metrics)
+
+## Monitoring app view
+
+The Monitoring app view provides aggregated statistics across all functions.
+The lag and commited offset statistics reflect the current state of the streams. 
+
+<img src="../_static/images/monitor-apps.png">
+<br>
+The <b>tiles</b> at the top present:
+
+- Applications: The total number of monitoring applications.
+- App Status: The number of functions running and the failed functions.
+- Endpoints: The total number of model endpoints, categorized by Batch and Real-time
+- Running Interval: The interval at which the apps monitor the models. 
+
+The <b>graph</b> shows the model endpoints with suspected/detected issues. The time interval for each bar is based on the running interval:
+- Up to 6 hours: 10-minute intervals
+- 2–72 hours: 1-hour intervals
+- More than 72 hours: 1-day intervals
+
+<b>System functions</b> shows the status of the three applications that support model monitoring: Controller, Writer, Reader.
+See more about these functions in {ref}`model-monitoring-des`.
```

Comment:
are you sure the reference is valid? I see it as {ref}`model-monitoring-des`
I guess it should be [model-monitoring-architecture] (#model-monitoring-des)